sdk/easm/Azure.Analytics.Defender.Easm/src/Azure.Analytics.Defender.Easm.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This is the Easm client library for developing .NET applications with rich experience.</Description>
```

Comment:
This is the description that will show up in the NuGet overview.   We should consider updating this, as the current wording doesn't tell someone unfamiliar with the library what it should be used for or what it is.